rpm/kubelet.spec

```diff
@@ -185,6 +189,9 @@ mv cni-plugins/bin/ %{buildroot}/opt/cni/
 
 
 %changelog
+* Tue Sep 25 2018 Chuck Ha <chuck@heptio.com> - 1.11.0
```

Comment:
1.12.1?

Reply:
I think I am confused about the version of cri-tools. I might defer this fix to @feiskyer. It looks like cri-tools matches the major/minor but not the patch.

Reply:
for example, k8s v1.11.3 depends on cri-tools v1.11.1, but k8s v1.12.x should match cri-tools v1.12.<latest cri-tools patch release>

Reply:
I believe this is the changelog for the kubelet package, so it's referring to kubernetes versions here. it's definitely incomplete, though...

Reply:
oh, sure, I can change the version on this line.